app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_parts = command.split()
+    if len(command_parts) != 3:
+        raise ValueError(f"Expected 3 arguments, got {len(command_parts)}")
+
+    command, old_file, new_file = command_parts
+    if old_file == new_file:
+        raise ValueError(f"File {new_file} already exists")
+
+    try:
+        with open(old_file, "r") as file_in:
+            with open(new_file, "w") as file_out:
+                file_out.write(file_in.read())
+    except FileNotFoundError:
```

Comment:
OSError would cover most errors related to work with file, not only when file not found. Instead of reraising you can print that something went wrong during file handling